conf/application.conf

```diff
@@ -177,4 +177,5 @@ play.http.parser.maxMemoryBuffer = 20MB
 akka.http.parsing.max-content-length = 20MB
 play.server.akka.max-content-length = 20MB
 play.server.akka.http.parsing.max-content-length = 20MB
-ifs.timeout = 40s
\ No newline at end of file
+ifs.timeout = 40s
+ifs.cache.timeout = 15m
```

Comment:
I spoke with Sanjay and from what I can tell we only need to get this call once per session. Please advise if you think otherwise.